### PR TITLE
Invite tab: Your invites screen

### LINKED ISSUE #903: Make invite tabs “+25 more” clickable
show full list of people you invited, picture, when, etc

### PR DESCRIPTION
**Pull request by nalinbhardwaj**
simple your invites screen, closes #903 

image

### CODE REVIEW

File: apps/daimo-mobile/src/view/screen/InviteScreen.tsx [hunk 1]
```diff
@@ -99,7 +102,14 @@ function InviteesBubbles({ invitees }: { invitees: EAccount[] }) {
       {moreInvitees > 0 && (
         <>
           <Spacer w={8} />
-          <TextBody color={color.grayMid}>+{moreInvitees} more</TextBody>
+          <Pressable
+            onPress={() => nav.push("YourInvites")}
+            children={({ pressed }) => (
```
Comment (dcposch) [inline apps/daimo-mobile/src/view/screen/InviteScreen.tsx:107]: recommend adding `hitSlop={32}` or so to make this easier to tap

File: apps/daimo-mobile/src/view/screen/InviteScreen.tsx [hunk 2]
```diff
@@ -127,18 +137,25 @@ function HeaderCountText({
...
       {showInviteesCount && (
-        <TextCenter>
-          <TextBody color={color.primary}>
-            You've invited {invitees?.length}{" "}
-            {invitees?.length === 1 ? "friend" : "friends"}
-          </TextBody>
-        </TextCenter>
+        <Pressable
+          onPress={() => nav.push("YourInvites")}
+          children={({ pressed }) => (
```
Comment (dcposch) [inline apps/daimo-mobile/src/view/screen/InviteScreen.tsx:150]: same here you can try hitSlip 16 or 32 to see what feels right

**Review by dcposch — APPROVED** [review]
LGTM after comments

File: apps/daimo-mobile/src/view/screen/YourInvitesScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    <View style={{ marginHorizontal: 16 }}>
+      <TouchableHighlight
+        onPress={() => navToAccountPage(invitee, nav)}
+        {...touchHighlightUnderlay.subtle}
+        style={styles.rowUnderlayWrap}
+      >
+        <View style={styles.inviteeRow}>
+          <View style={styles.inviteeRowLeft}>
+            <TouchableOpacity onPress={() => navToAccountPage(invitee, nav)}>
+              <ContactBubble contact={{ type: "eAcc", ...invitee }} size={36} />
```
Comment (dcposch) [inline apps/daimo-mobile/src/view/screen/YourInvitesScreen.tsx:53]: if the whole row already navs to the account page, then i don't think that the contact bubble needs to be individually tappable

File: apps/daimo-mobile/src/view/screen/InviteScreen.tsx (hunk 2 above)
Comment (nalinbhardwaj) [reply apps/daimo-mobile/src/view/screen/InviteScreen.tsx:150]: 16 feels good to me, done

File: apps/daimo-mobile/src/view/screen/YourInvitesScreen.tsx (hunk 3 above)
Comment (nalinbhardwaj) [reply apps/daimo-mobile/src/view/screen/YourInvitesScreen.tsx]: done

**Pull request merged**